content/messaging/agent-chat-api/changelog/index.mdx

```diff
@@ -9,6 +9,12 @@ desc: "Changelog of the Agent Chat API"
 
 This document is the record of all the changes in the **Agent Chat API**, Web and RTM references, starting from **version 3.0**.
 
+## [v3.3] - 2020-06-30 - Developer preview
+
+### Events
+
+- There's a new parameter `alternative_text` for images within **Rich Messages** ([Web](/messaging/agent-chat-api/v3.3/#rich-message) & [RTM](/messaging/agent-chat-api/v3.3/rtm-reference/#rich-message))
```

Comment:
One of the rules of the new format is to start with the method/data structure/push/webhook name so that it's easier to scan the block of text. I suggest the following:

> The **Rich Message** ([Web](/messaging/agent-chat-api/v3.3/#rich-message) & [RTM](/messaging/agent-chat-api/v3.3/rtm-reference/#rich-message)) data structure has a new parameter, `alternative_text`.

Reply:
Agreed on the final form outside of github, applied